org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/xtext/index/BuildCycleManager.xtend

```diff
@@ -168,13 +214,11 @@ interface ChangeDetector {
 
 }
 
+@Accessors(PUBLIC_GETTER)
 class ChangedResources {
 
-	val modifiedResources = <URI>newHashSet
-	val deletedResources = <URI>newHashSet
-
-	def Set<URI> getModifiedResources() { return modifiedResources }
-
-	def Set<URI> getDeletedResources() { return deletedResources }
+	val Set<URI> modifiedResources = <URI>newHashSet
```

Comment:
Is the generic parameter to newHashSet really necessary? Thought that xtend could derive this from the l-value of the assignment.